src/main/kotlin/main.kt

```diff
@@ -16,6 +18,9 @@ fun main() {
     val jsonGenerator = JsonGenerator()
     val serverError = InternalServerError()
     val fileIo = FileIo()
+    val filepath = "src/main/kotlin/resources/task-list.txt"
+    val todoList = ToDoList(filepath, fileIo, jsonGenerator)
+    val todo = ToDo(todoList)
```

Comment:
I think it's a bit strange that a to-do knows about the list of to-dos. 

Did you mention today that you were going to get rid of one of these classes? I think you might not need a `ToDo` class yet.

Reply:
Yes, in the next PR I have removed `ToDo`. I realized it was an early abstraction.